pom.xml

```diff
@@ -225,7 +226,7 @@
           <plugin>
             <groupId>org.owasp</groupId>
             <artifactId>dependency-check-maven</artifactId>
-            <version>3.2.1</version>
+            <version>4.0.2</version>
```

Comment:
This seems unrelated, as well as the exclusions updates, maybe best if it goes in a different PR?